lib/cdo/scripts/script_info.rb

```diff
@@ -0,0 +1,54 @@
+# Info on existing tutorial scripts on Code Studio,
+# sometimes referenced in Pegasus.
+# Used for conditional behaviors.
+
+class ScriptInfo
+  HOC_2013_NAME = 'Hour of Code' # this is the old (2013) hour of code
```

Comment:
The risk here is is that we change the name of a script and break caching configuration (since the name is a key.)  I have eyeballed all of the scripts we care about and verified the names are the same.